services/datalad/datalad_service/common/git.py

```diff
@@ -78,9 +81,22 @@ def git_commit(repo, file_paths, author=None, message="[OpenNeuro] Recorded chan
             'HEAD points at invalid branch name ({}) and commit was aborted'.format(repo.references['HEAD'].target))
     # Refresh index with git-annex specific handling
     annex_command = ["git-annex", "add"] + file_paths
-    subprocess.run(annex_command, check=True, cwd=repo.workdir)
-    repo.index.add_all(file_paths)
-    repo.index.write()
+    try:
+        subprocess.run(annex_command, check=True, cwd=repo.workdir)
```

Comment:
I think you need this to access stdout/stderr below. I'm not looking at the docs right now, though.

```suggestion
        subprocess.run(annex_command, check=True, capture_output=True, cwd=repo.workdir)
```